data.md

```diff
@@ -604,6 +627,11 @@ endmodule
 module WASM-DATA-CONCRETE  [concrete]
    imports WASM-DATA-COMMON
 
+   syntax Int ::= #signedTotal ( IValType , Int)  [function, total, klabel(#signedTotal), symbol, smtlib(signedTotal)]
```

Comment:
I was planning to use it for rules that use `#signed` in the RHS, because partial functions make the Booster backend fall back to the Haskell backend, replacing `#signed` with `#signedTotal` and adding `definedSigned` in the requires clause.

Since I'm not using it yet, I'll remove it.